core/controllers/dashboard.py

```diff
@@ -257,6 +258,204 @@ def _round_average_ratings(rating):
         self.render_json(self.values)
 
 
+class LearnerDashboardPage(base.BaseHandler):
```

Comment:
I could create a new file by the name learner_dashboard.py and keep the learner dashboard functionality over there. But changing all the function names from `dashboard` to `creator dashboard` seems like a pretty big refactor. Functions by the name dashboard have been used extensively throughout the codebase and even comments - DashboardPage, NotificationsDashboardHandler, DashboardHandler. Instead, can we have a convention like this - just dashboard by default refers to the creator dashboard and those starting with learner refer to the learner dashboard. What do you think of this?